README.md

```diff
@@ -1 +1,28 @@
-# bindle
\ No newline at end of file
+# bindle
+
+Public build scripts.
```

Comment:
Should we allude to the fact we have a private build scripts?

perhaps something like
```suggestion
Build tool scripts.
```